### PR TITLE
(PCP-192) Add in-reply-to to envelope schema

### PR DESCRIPTION
**Pull request by parisiale**
Adding optional entry in the envelope content.
Specifying that inventory_response messages must include such entry.

### CODE REVIEW

File: pcp/inventory.md [hunk 1]
```diff
@@ -51,6 +52,9 @@ An inventory response message is decribed by the following json-schema:
 }
 ```
 
+The envelope `content` of inventory response messages must include the optional
+`in-reply-to` entry, set to the `id` of the inventory request message.
```
Comment (ploubser) [inline pcp/inventory.md]: Maybe reword this bit? "must include the optional" doesn't sound right.

Comment (parisiale) [inline pcp/inventory.md]: It does sound correct to my foreign ears, but I'll remove "the optional" (only 16 results in Google Scholar, I surrender).

File: pcp/message.md [hunk 2]
```diff
@@ -44,7 +45,8 @@ JSON schema:
 | expires | string | time entry in ISO8601 format indicating when the message is no longer valid anymore
 | targets | array | list of PCP URIs (string items) indicating the recipients
 | sender | string | PCP URI indicating the endpoint node (broker or client node) that sent the message
-| destination_report | bool | If set to true broker must respond with a list of URI's that include all targets for a message |
+| destination_report | bool | if set to true broker must respond with a list of URI's that include all targets for a message |
+| in-reply-to | string | used by the broker to indicate the ID of the received message to which he's replying |
```
Comment (richardc) [inline pcp/message.md]: This isn't only for the broker, nor is the broker a human male.
